File: MyAnimeListJava/src/main/java/com/example/myanimelistjava/models/enums/Status.java
package com.example.myanimelistjava.models.enums;

public enum Status {
    FINISHED_AIRING("Finished Airing"),
    CURRENTLY_AIRING("Currently Airing"),
    NOT_YET_AIRED("Not yet aired");

    private final String value;

    Status(String value) {
        this.value = value;
    }

    public String getValue() {
        return value;
    }
}

File: MyAnimeList/src/main/kotlin/com/example/myanimelist/models/enums/Status.kt
package com.example.myanimelist.models.enums

import javafx.collections.FXCollections
import javafx.collections.ObservableList

enum class Status(val value: String = "Unknown") {
    FINISHED_AIRING("Finished Airing"),
    CURRENTLY_AIRING("Currently Airing"),
    NOT_YET_AIRED("Not yet aired");

    companion object {
        val sample: ObservableList<String> =
            FXCollections.observableArrayList(Status.values().map { it.value }.toList())
    }
}
